src/yunohost/certificate.py

```diff
@@ -809,6 +816,15 @@ def _domain_is_accessible_through_HTTP(ip, domain):
     return True
 
 
+def _domain_is_resolved_locally(domain):
+    try:
+        requests.head("http://%s/" % domain)
```

Comment:
Fixed in 86f97d6

NB : I used `socket.gethostbyname()` because it's seems closer to what `requests` uses than `dns.resolve`... `dns.resolve` will perform a 'full' DNS request and return the public ip, but if the domain is in `/etc/hosts`, `requests` will actually use what's in `/etc/hosts`... `socket.gethostbyname()` takes `/etc/hosts` into account.